TP/Frontend/src/app/components/product-detail/product-detail.component.scss

```diff
@@ -0,0 +1,206 @@
+// arreglar esta linea.
+// si lo saco, se me ve un recuadro horrible. (corregir)
+.container {
+  outline: none !important;
```

Comment:
no hay que sacar los outlines, son ayudas para gente con discapacidad, miren esto https://medium.com/better-programming/a11y-never-remove-the-outlines-ee4efc7a9968